package.json

```diff
@@ -46,11 +46,14 @@
     "eslint": "^3.12.2",
     "eslint-plugin-import": "^2.2.0",
     "karma": "^1.3.0",
+    "karma-mocha": "^1.3.0",
```

Comment:
@aqrln yeah, completely forgot about npm commands =). Got used to editing config files manually that much